content/docs/instrumenting/exporters.md

```diff
@@ -53,6 +53,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [RabbitMQ exporter](https://github.com/kbudde/rabbitmq_exporter)
    * [RabbitMQ Management Plugin exporter](https://github.com/deadtrickster/prometheus_rabbitmq_exporter)
    * [Mirth Connect exporter](https://github.com/vynca/mirth_exporter)
+   * [Kafka consumer group exporter](https://github.com/kawamuray/prometheus-kafka-consumer-group-exporter)
```

Comment:
Sorry for taking so long.

> Keep it sorted.

Force-pushed a fix for this.

> It'd also be good in your readme to point to the jmx exporter for all the other kafka stats

Good idea. I'm not the maintainer of the repository, but I've submitted [a PR](https://github.com/kawamuray/prometheus-kafka-consumer-group-exporter/pull/12) to get that added.